forum/qa-plugin/user-activity-log/user-activity-log.php

```diff
@@ -0,0 +1,139 @@
+                    ),
+                    
+                    'filter' => array(
+                        'type' => 'select',
+                        'tags'=>'name=condition value="'.$this->condition.'"',
+                        'options' => [
+                                'username' => qa_lang('user-activity-log/username'),
+                                'type' => qa_lang('user-activity-log/eventType'),
+                        ]
+                        ,
+                        'label' => qa_lang_html('user-activity-log/filters'),
+                    ),
+
+                    'date' => array(
+                        'label' => qa_lang_html('user-activity-log/date'),
+                        'tags' => 'placeholder=YYYY-MM-DD name=date value="'.$this->date.'"',
+                    ),
+
+                    'fromOldest' => [
+                        'type' => 'select',
+                        'tags'=>'name=fromOldest value="'.$this->sortBy.'"',
+                        'options' => [
+                                0 => qa_lang('user-activity-log/oldest'),
+                                1 => qa_lang('user-activity-log/newest'),
+                        ]
+                        ,
+                        'label' => qa_lang_html('user-activity-log/from'),
+                    ],
+
+                    'resultsCount' => array(
+                        'label'=> qa_lang_html('user-activity-log/count'),
+                        'tags'=> 'name=resultsCount value="'.$this->resultsCount.'"',
+                    )
+                ),
+
+                'buttons' => array(
+                    'submit' => array(
+                        'tags' => 'name="search"',
+                        'label' => qa_lang_html('user-activity-log/search-label'),
+                        'value' => '1',
+                    ),
+                ),
+            );
+
+            if(qa_get_logged_in_level() >= QA_USER_LEVEL_MODERATOR){
+                $qa_content['form']['fields']['filter']['options']['ip'] = qa_lang_html('user-activity-log/ipAdress');
+            }
+            $qa_content['navigation']['sub']= qa_admin_sub_navigation();
+            $qa_content['custom_2'] = 
+            '<div class = "modal-content">
+                <h1>'.qa_lang_html('user-activity-log/EventModalHeader').'</h1>
+                <p>
+                    <table>
+                        <tr>
+                            <th>'.qa_lang_html('user-activity-log/users').'</th>
+                            <th>'.qa_lang_html('user-activity-log/questions').'</th>
+                            <th>'.qa_lang_html('user-activity-log/answers').'</th>
+                        </tr>
+                        '.$this->genereateEventsTable().'
+                        </table>
+                </p>
+                <a href = "https://docs.question2answer.org/plugins/modules-event/">'.qa_lang_html('user-activity-log/more').'</a>
+                <br/>
```

Comment:
Czemu tutaj służy `<br/>`? Jeśli trzeba coś stylistycznie odsunąć w pionie, to do tego służy CSS (np. `padding` lub `margin`, albo [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)/[Grid](https://css-tricks.com/snippets/css/complete-guide-grid/)).

Reply:
Tutaj ten `<br/>`  służy do przeniesienia przycisku zamknięcia do następnej lini``

Reply:
Poprawione